CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This log is intended to keep track of package changes, including
 but not limited to API changes and file location changes. Minor behavioral
 changes may not be included if they are not expected to break existing code.
 
+## 0.3.10 (2023-02-21)
+
+- Add support for skipHeaderRow as a customization override
```

Comment:
I would word this a little differently since it's only an update to the typescript definitions... like 'Update Typescript definitions for `skipHeaderRow` in customization overrides' or something..